src/test/java/io/strimzi/kafka/bridge/mqtt/kafka/BridgeKafkaProducerServiceTest.java

```diff
@@ -13,29 +13,33 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
+
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.when;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
 
 /**
- * Unit tests for {@link BridgeKafkaProducer}
+ * Unit tests for {@link BridgeKafkaProducerService}
  */
-public class BridgeKafkaProducerTest {
+public class BridgeKafkaProducerServiceTest {
 
     /**
-     * Test the {@link BridgeKafkaProducer#send(ProducerRecord)} method
+     * Test the {@link BridgeKafkaProducerService#send(ProducerRecord)}} method
      */
     @Test
-    public void testProducerNoAck() {
+    public void testProducerSend() {
```

Comment:
It's not totally clear how this test would work even because you are doing the sending with and without ACK together. It would be better to have two different tests to simplify the scenarios.

Reply:
I was thinking about this.
But for noAck send, I do not think there are much to test.

```java
doNothing().when(producer).sendNoAck(any(ProducerRecord.class));
producer.sendNoAck(record);
```
and then what should be next?